style.css

```diff
@@ -27,8 +28,29 @@ body {
   margin: 0;
   padding: 0;
   list-style-type: none;
+ }
+
+.topic li 	{
+	margin: 10px auto;
+	width: 900px;
+}
+
+.topic-title {
+	color: dark blue;
```

Comment:
`dark blue` isn't actually a value but `darkblue` is. If you use the inspector in your browser you can actually edit the CSS and it will autocomplete the available colour names for you (no one remembers them all). My preference is usually to just stick to hex codes (in this case `#000080`)